libs/librrgraph/src/io/rr_graph_uxsdcxx_serializer.h

```diff
@@ -460,11 +460,13 @@ class RrGraphSerializer final : public uxsd::RrGraphBase<RrGraphContextTypes> {
         //
         // If the switch name is not present in the architecture, generate an
         // error.
+        // If the graph is written when flat-routing is enalbed, the types of the switches inside of the rr_graph are also
+        // added to the XML file. These types are not added the data structure that contain arch switch types. It's remained
```

Comment:
Keeping this code or not? (Can give an error instead if the arch file doesn't line up).
enalbed --> enabled, 
It's remained --> It remains a future